2.5/README.md

```diff
@@ -0,0 +1,141 @@
+Ruby 2.5 Docker image
```

Comment:
No references to docker should be present in user-facing texts.

Reply:
Please use `container` instead

Reply:
OK I will modify referring 2.4/README.md.